src/components/DxpFacilitySwitcher.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title>
+        {{ $t('Facility') }}
+      </ion-card-title>
+    </ion-card-header>
+    <ion-card-content>
+      {{ $t('Specify which facility you want to operate from. Order, inventory and other configuration data will be specific to the facility you select.') }}
+    </ion-card-content>
+    <ion-item lines="none">
+      <ion-label>
+        {{ currentFacility.facilityName }}
+        <p>{{ currentFacility.facilityId }}</p>
+      </ion-label>
+      <ion-button id="open-facility-modal" slot="end" fill="outline" color="dark">{{ $t('Change')}}</ion-button>
```

Comment:
this button should only be shown if the user is associated with more than one facility. @R-Sourabh 

<img width="615" alt="Screenshot 2025-01-01 at 5 54 07 PM" src="https://github.com/user-attachments/assets/f56f243a-3bf2-4834-ba9c-1312f433b66b" />